common.sh

```diff
@@ -138,7 +142,32 @@ function build_kb {
     opts=-ldflags "-X sigs.k8s.io/kubebuilder/cmd/version.kubeBuilderVersion=$INJECT_KB_VERSION"
   fi
 
-  go build $opts -o $tmp_root/kubebuilder/bin/kubebuilder ./cmd
+  GO111MODULE=on go build $opts -o $tmp_root/kubebuilder/bin/kubebuilder ./cmd
+}
+
+function fetch_go_tools {
+  header_text "Checking for dep"
+  if ! is_installed dep ; then
+    header_text "Installing dep"
+    DEP_DIR=$(go env GOPATH)/src/github.com/golang/dep
+    mkdir -p $DEP_DIR
+    pushd $DEP_DIR
+    git clone https://github.com/golang/dep.git .
+    DEP_LATEST=$(git describe --abbrev=0 --tags)
+    git checkout $DEP_LATEST
+    mkdir bin
+    go build -ldflags="-X main.version=$DEP_LATEST" -o bin/dep ./cmd/dep
+    export PATH=$(go env GOPATH)/src/github.com/golang/dep/bin:$PATH
```

Comment:
this is what I changed, sorry I squashed before letting you review. `go install` seemed to be a bit finnicky between prow/travis, and with the changes to GOPATH later I figured I should force it to a static location.

Reply:
yeah, that's fine